scrapinghub/client/collections.py

```diff
@@ -91,7 +91,7 @@ def list(self):
 
         :return: a list of collections where each collection is
             represented by a dictionary with ('name','type') fields.
-        :rtype: list[dict]
+        :rtype: list of dicts
```

Comment:
I think you should follow Sphinx examples for container types like [here](http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists) - which means keeping `list[dict]` like it was before.

... or typing notation https://docs.python.org/3/library/typing.html